src/main/java/org/apache/maven/model/jdom/JDomDependencies.java

```diff
@@ -64,7 +61,8 @@ public JDomDependencies(Element jdomElement) {
 
   @Override
   public boolean add(Dependency dependency) {
-    return addInternal(dependency, -1);
+    add(size(), dependency);
+    return true;
```

Comment:
Please return the return value of the add method.

Reply:
This behaves exactly like the JDK List implementations - see implementations of overridden methods. The code here is an exact copy of the default `AbstractList#add(E)` implementation - `AbstractList#add(int, E)` only returns `void`.